### ISSUE TITLE
Hierarchy isn't updating when Graphs inside are modified

### ISSUE THREAD

**Issue by JulienTrue** [JirkaDellOro/FUDGE#409]
Having several nested graphs and then adding components in one of them, will not correctly update and display on the nodes/graphs further up in the hierarchy.

**Comment by JirkaDellOro** [comment]
I think at this point of time, structural changes to graphs or graph-instances will not sync with other instances, only changes of values. Is it this bevavior you notice?

For now, to change a graph structurally, you need to edit the graph (not an instance), save and then reload the project, to recreate the instances. Does that work?

**Comment by JulienTrue** [comment]
Indeed, structural changes to graphs do not sync with other instances, but value changes do!

And yes, saving and reloading the project always recreates the instances which is the current work flow.
